src/services/CertificateGenerationService.ts

```diff
@@ -191,9 +191,26 @@ class CertificateGenerationService {
       Signature: {
         ImageType: "png",
         ImageData: signature,
```

Comment:
Does this need modifying so that a `signature` isn't sent in the case of an amendment as the ticket says the certificate reissuer would hand sign it?

Reply:
Good spot.
It's a bit ambiguous in the ticket. We default to hand writing if the signature is not found in S3. However, I can see that the code still tries to get the signature for the tester. I'll amend so it tries for the amender too.

Reply:
Best case theres a signature there for the person and they dont need to sign it again but most often then not there wont be and a manual one will be needed. Is what im thinking